launch/client.py

```diff
@@ -312,6 +319,7 @@ def create_model_bundle(  # pylint: disable=too-many-statements
                 model_bundle_name,
             )
 
+        cloudpickle.register_pickle_by_value(launch)  # register launch API
```

Comment:
btw on the individual endpoint side we do have a `launch-python-client` dependency pinned to a (at this point kinda old) git hash (see `hosted_model_inference/hosted_model_inference/inference/requirements_base.txt`)

Reply:
oh, then probably this should be deleted in the next versions...